test/integration/controller/jobset_controller_test.go

```diff
@@ -635,39 +674,45 @@ func numExpectedServices(js *jobset.JobSet) int {
 func completeAllJobs(jobList *batchv1.JobList) {
 	ginkgo.By("completing all jobs")
 	for _, job := range jobList.Items {
-		completeJob(&job)
+		updateJobStatusConditions(&job, batchv1.JobStatus{
+			Succeeded: int32(len(jobList.Items)),
```

Comment:
why is succeeded pods equal to number of jobs?

Reply:
removed it as unused